models/article.model.js

```diff
@@ -28,3 +28,16 @@ exports.articleByID = (article_id) => {
       return rows[0];
     });
 };
+
+exports.checkArticleExists = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
```

Comment:
It's not an error if there are no comments for an article. You are missing a 200 test to check for this.